utils/frame/benchmarking-cli/src/shared/mod.rs

```diff
@@ -66,6 +67,50 @@ where
 	s
 }
 
+/// A Handlebars helper to add an underscore after every 3rd character,then format it,
+/// i.e. a separator for large numbers : 123000000 -> 123_000 * WEIGHT_PER_NANOS.
+#[derive(Clone, Copy)]
+pub struct WeightAsNanoseconds;
+
+impl handlebars::HelperDef for WeightAsNanoseconds {
+	fn call<'reg: 'rc, 'rc>(
+		&self,
+		h: &handlebars::Helper,
+		_: &handlebars::Handlebars,
+		_: &handlebars::Context,
+		_rc: &mut handlebars::RenderContext,
+		out: &mut dyn handlebars::Output,
+	) -> handlebars::HelperResult {
+		let param = h.param(0).unwrap();
+		let weight_as_nanoseconds_param = weight_as_nanoseconds(param.render());
+		out.write(&weight_as_nanoseconds_param)?;
+		Ok(())
+	}
+}
+
+/// Add an underscore after every 3rd character, then format it, i.e. a separator for large numbers:
+/// 123000000 -> 123_000 * WEIGHT_PER_NANOS.
+fn weight_as_nanoseconds<Number>(i: Number) -> String
+where
+	Number: std::string::ToString,
+{
+	let mut i = i.to_string().parse::<u64>().unwrap();
+	i = i.checked_div(constants::WEIGHT_PER_NANOS).unwrap();
+
+	let mut s = String::new();
+	let i_str = i.to_string();
+	let a = i_str.chars().rev().enumerate();
+	for (idx, val) in a {
+		if idx != 0 && idx % 3 == 0 {
+			s.insert(0, '_');
+		}
+		s.insert(0, val);
+	}
+	s.insert(0, '(');
+	s.push_str(" as Weight).saturating_mul(WEIGHT_PER_NANOS)");
```

Comment:
You could also do this in the template instead and then change the `standard error` comment to:  
```rust
// Standard Error: {{weightAsNanoseconds cw.error}} NS
```
otherwise it reads a bit awkward like:  
```rust
// Standard Error: (5 as Weight).saturating_mul(WEIGHT_PER_NANOS)
```